finalPrep/README.md

```diff
@@ -18,37 +18,43 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [X] I have refactored my `section1` exercises to the best of my ability
+- [X] I have refactored my `section2` exercises to the best of my ability
+- [X] I have refactored my `section3` exercises to the best of my ability
+- [X] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [X] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [X] Study/Project work time
+- [X] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="1440" alt="Screen Shot 2021-08-07 at 6 43 05 PM" src="https://user-images.githubusercontent.com/78453792/128618929-55288abf-f61d-44d1-9c2a-f878825de997.png">
```

Comment:
Nice job overall here. It looks like you plan on burning the midnight oil with homework. While I had plenty of late nights developing, I encourage you to be mindful of how focused you can stay, and until when, to work most efficiently. Turing days are long, and it could prove hard to get back into work that late. I also suggest breaking up longer work blocks with some 15 min breaks here and there- try to take stock of the daily activities that keep you sane, and get them in your calendar to help you remember to do those things! Also, dedicated time to review the past week, take honest stock of how you did with your calendar, and make adjustments for the upcoming week is an excellent practice to get into.